app/assets/javascripts/listings/components/listing/eligibility-section.html.slim

```diff
@@ -77,7 +78,7 @@ accordion-heading.lead
           p.custom-counter_text ng-if="::!$ctrl.listingHasOnlySROUnits($ctrl.parent.listing) && !$ctrl.parent.listingHasSROUnits($ctrl.parent.listing)"
             span ng-if="::$ctrl.parent.listingIs('Transbay Block 7', $ctrl.parent.listing)"
               | {{'LISTINGS.OCCUPANCY_SEE_UNIT_INFO' | translate}}
-            span ng-if="::!$ctrl.parent.listingIs('Transbay Block 7', $ctrl.parent.listing)"
+            span ng-if="::!$ctrl.parent.isSale($ctrl.parent.listing) && !$ctrl.parent.listingIs('Transbay Block 7', $ctrl.parent.listing)"
```

Comment:
This is a hacky way of not showing any text for sales in the occupancy section. We may choose to change the text on Monday and then I can add it to locale-en.json

Reply:
We're going to keep this text as is